airqo-platform/AirQo-frontend
# [Netmanager] Apply airqlouds summary api to airqloud registry

#### Summary of Changes (What does this PR do?)

- Fetch airqlouds data from summary endpoint so as to load fast
- Refactor approach of getting airqloud details without loading all airqlouds

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Visit the airqloud registry table, the page should load faster than current production
- Select an airqloud which has sites, the details should load also much faster than current production

#### What are the relevant tickets?

- [AN-408](https://airqoteam.atlassian.net/browse/AN-408)

[AN-408]: https://airqoteam.atlassian.net/browse/AN-408?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
